cli/command/node/health.go

```diff
@@ -2,16 +2,28 @@ package node
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
 	"time"
 
 	"github.com/dnephin/cobra"
 
+	"github.com/storageos/go-api/types"
 	"github.com/storageos/go-cli/cli"
 	"github.com/storageos/go-cli/cli/command"
 	"github.com/storageos/go-cli/cli/command/formatter"
 	cliTypes "github.com/storageos/go-cli/types"
 )
 
+var (
```

Comment:
Is there a good reason not to just use the defaults in the API, rather than copy-pasting?

Reply:
Nope, no reason, just failed to realize 😅